jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -131,7 +132,8 @@ public Job build() {
                 _name = "cookjob";
             }
```

Comment:
I will set the default. The Boolean is to be consistent with others. I could address those in other CR.